features/view_specs/view_spec.feature

```diff
@@ -1,6 +1,55 @@
 Feature: view spec
 
-  View specs live in spec/views and render view templates in isolation.
+  View specs should be placed in `spec/views`,
+  and verify the content of view templates in isolation
+  (that is, without invoking a controller).
+
+  Overview
+  --------
+
+  View specs generally follow three steps:
+
+  ```ruby
+  assign(:widget, Widget.new)  # sets @widget = Widget.new in the view template
+
+  render
+
+  expect(rendered).to match(/text/)
+  ```
+
+  1. Use the `assign` method to set instance variables in the view.
+     Technically, `@widget = Widget.new` would work too,
+     but RSpec doesn't officially support this pattern.
+     (It only works as a side effect of the fact that
+     view specs include `ActionView::TestCase` behavior.
+     Be aware that it may be made unavailable in the future.)
+
+  2. Use the `render` method to render the view.
+
+  3. Set expectations against the resulting `rendered` object.
+
+  Notes
+  -----
+
+  * To apply a layout to the view template being rendered,
+    be sure to specify both the template and layout explicitly:
+
+    ```ruby
+    render :template => "events/show", :layout => "layouts/application"
+    ```
+
+  * View specs expose a `controller` object,
+    which can be used to set expectations about the route (path + parameters)
```

Comment:
Its not intended to test the path or parameters, but to allow the view to use that information for various things.